modules/atlas/src/main/resources/schemas/neurosciencegraph/atlas/transformation/v0.1.0.json

```diff
@@ -0,0 +1,62 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/atlas/transformation/v0.1.0/shapes/"
+    }
+  ],
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.0"
```

Comment:
Use the [neurosciencegraph/commons/activity/v0.1.3](https://github.com/INCF/neuroshapes/blob/master/modules/commons/src/main/resources/schemas/neurosciencegraph/commons/activity/v0.1.3.json) instead

Reply:
Done